### DISCUSSION TITLE
Search field on entries page should inform its limitations

### DISCUSSION THREAD

**Discussion by robsonsobral** [ExpressionEngine/ExpressionEngine#2910] [Feature Requests]
The search field requires at least 3 characters, but there's no mention of it on the interface, which confuses users.

![image](https://user-images.githubusercontent.com/931642/131569450-0a00a42c-62ed-4ac2-818a-7b1061f99a15.png)

**Comment by robsonsobral** [comment]
May I add some info on the placeholder, @ajohnson ? Or to enable a HTML validation!

**Comment by ajohnson** [comment]
@robsonsobral This is a tricky one. I don't think we should put that kind explicit info within the placeholder itself, especially since during many searches the character requirement will be a non-issue.

But I can see using something similar to EE's standard validation message placement under the Search input to communicate the requirement. It could look something like this:
<img width="509" alt="Screen Shot 2022-03-10 at 5 36 59 PM" src="https://user-images.githubusercontent.com/194579/157767782-da4d55f5-cbab-4e4d-b3b8-90ed65366fe0.png">

But the message should only appear AFTER the existing JS "watch" period has expired and the 3 character requirement has still not been met.

In other words, the validation message should NOT appear immediately. There should be a gracious delay between when the first character is typed and when the message finally appears. Otherwise the message could be very annoying when someone is just taking a quick, natural pause while in the process of typing their search criteria.

Does that make sense? Do you want to attempt a working example the team could look at?

**Comment by robsonsobral** [comment]
Your suggestion is perfect for me.
